prisma/seed.ts

```diff
@@ -0,0 +1,48 @@
+import { PrismaClient } from '@prisma/client';
+import * as bcrypt from 'bcrypt';
+import { faker } from '@faker-js/faker';
+
+const prisma = new PrismaClient();
+
+faker.seed(1);
+const SALT_ROUNDS = 10;
+
+const DEFAULT_PASSWORD = 'Password1';
+const DEFAULT_USERS_AMOUNT = 5;
+
+const hashPassword = async (password: string): Promise<string> => {
+  const salt = await bcrypt.genSalt(SALT_ROUNDS);
+
+  return bcrypt.hash(password, salt);
+}
+
+const seed = async () => {
+  try {
+    const hashedPassword = await hashPassword(DEFAULT_PASSWORD);
+
+    await prisma.$transaction(async (prisma) => {
+      for (let i = 0; i < DEFAULT_USERS_AMOUNT; i++) {
+        const email = faker.internet.email();
+
+        await prisma.user.upsert({
+          where: { email: email },
```

Comment:
I don't know if you cannot do this: where: { email }

Reply:
Yes. Fixed